docs/design/StratisSoftwareDesign.lyx

```diff
@@ -990,7 +990,30 @@ Security Policy
 \end_layout
 
 \begin_layout Standard
-Stratis D-Bus methods require root permissions.
+Most stratisd D-Bus methods require root permissions.
+ However, listing operations do not; this can be done by an unprivileged
+ user.
+ These permissions can be changed in the policy file included in the repo,
```

Comment:
I realized I'ld like to make the last three sentences more general...and also to avoid mention of GitHub. Suggested wording:

"The default permissions are specified in the policy file, stratisd.conf, included in the stratisd distribution. These defaults permit all actions by root users but restrict unprivileged users to read-only actions. Systems administrators can adjust permissions by editing the stratisd D-Bus policy files."

You never know...systems administrators on some systems might want to make administering the daemon wide open to anybody. I admit I can't think of any good reason for that right now...

Reply:
Resolved.